src/NuGet.Core/NuGet.Commands/PackagesLockFileBuilder.cs

```diff
@@ -16,7 +16,7 @@ public class PackagesLockFileBuilder
     {
         public PackagesLockFile CreateNuGetLockFile(LockFile assetsFile)
         {
-            var lockFile = new PackagesLockFile();
+            var lockFile = new PackagesLockFile(PackagesLockFileFormat.PackageReferenceVersion);
```

Comment:
Split the versions to allow the package ref projects to evolve the schema without impact on PC projects. Depending on the resolution of [issue](https://github.com/NuGet/Home/issues/9370) this may change. I propose to not block the current PR on the resolution of the issue mentioned.

Reply:
The version is currently used only by the PC validation that is not changed anyway. I propose to chnage the version only after the [issue](https://github.com/NuGet/Home/issues/9370)  has a resolution. I am saying this because based on the resolution of that issue the implementation may change. 

@zivkan , @nkolev92 what do you think?

Reply:
As long we fix it before the stable release, it's ok.

Reply:
Places I worked before coming to Microsoft often put things on the backlog to do later, but were forgotten about, or took much, much longer to implement that intended at the time the work was put off. Given we work on a shipping product, not an internal tool, it makes me nervous to say that that this user experience issue is being delayed in order to help this PR get merged sooner. If we ship this PR's work before implementing a change for the newly created issue, what will take mean particularly with compatibility and efforts to avoid breaking customers?

Reply:
Updated.